src/components/core/geoChart/tests/geoChartDataSource.spec.ts

```diff
@@ -3,17 +3,46 @@ import mapboxgl from "mapbox-gl";
 import { createPushpinDataSource } from "../geoChartDataSource";
 import { IGeoData } from "../../../../interfaces/GeoChart";
 import { LOCATION_LNGLATS, SIZE_NUMBERS, COLOR_NUMBERS } from "../../../../../stories/data/geoChart";
+import GeoChartColorStrategy from "../../../visualizations/chart/colorStrategies/geoChart";
+import { DEFAULT_COLORS, DEFAULT_COLOR_PALETTE } from "../../../visualizations/utils/color";
 
 describe("createPushpinDataSource", () => {
+    const mockColorStrategy = new GeoChartColorStrategy(DEFAULT_COLOR_PALETTE, null, null, null, null, null);
```

Comment:
could we get rid of `null, null, null, null, null`?

Reply:
fixed

Reply:
I'm not sure which is better: old or new code :)))
What do you think?

Reply:
this look to better, it does not get rid of `null`. :))